srcpkgs/python3-pybind11/template

```diff
@@ -23,5 +21,5 @@ checksum=1eed57bc6863190e35637290f97a20c81cfe4d9090ac0a24f3bbf08f265eb71d
 
 post_install(){
 vlicense LICENSE LICENSE
-
   }
+
```

Comment:
Looks like you accidentally added an empty line at the end of the template.

Reply:
And the license should be [BSD-3-Clause](https://opensource.org/licenses/BSD-3-Clause)

Reply:
Fixed these, thank you.